packages/filter/src/Filter.tsx

```diff
@@ -67,12 +69,18 @@ const Filter = ({
   };
 
   return (
-    <Dropdown targetTestId={testId} targetClassName={dropdownClasses} {...props}>
+    <Dropdown
+      targetTestId={testId}
+      disabled={disabled}
+      targetClassName={dropdownClasses}
+      {...props}
+    >
```

Comment:
The dropdown can no longer be opened when the "disabled" prop is sent to `Filter` component.